inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp

```diff
@@ -1035,6 +1037,11 @@ void MKLDNNQuantizeNode::init() {
             }
 #endif
 
+            if (il > ih) {
+                THROW_IE_EXCEPTION << "Quantize layer with name '" << getName() << "' has unsupported input quantize ranges inputLow > inputHigh: "
+                                   << il << " > " << ih;
```

Comment:
minor: you need to print what these values mean. Smt like: "inputLow (5) > inputHigh (-5)"